pkg/controllers/kepler.go

```diff
@@ -167,13 +170,21 @@ func (r KeplerReconciler) updateStatus(ctx context.Context, req ctrl.Request, re
 			r.logger.V(6).Info("Reconcile has deleted kepler; skipping update")
 			return nil
 		}
+		now := metav1.Now()
 
 		if recErr != nil {
```

Comment:
What do you feel about adding the Daemonset status - NumberAvailable, Schedueld, etc here and modify the Kepler CRD to show readiness just like a pod? 
```
oc get keplers 

NAME                 DESIRED   CURRENT   READY   UP-TO-DATE   AVAILABLE   NODE SELECTOR            AGE                           kepler                            3         3         3       3            3           kubernetes.io/os=linux   41
```

Reply:
```
 ❯ kubectl get keplers

NAME     DESIRED   CURRENT   READY   UP-TO-DATE   AVAILABLE   NODESELECTOR
kepler   3         3         3       3            3           {"kubernetes.io/os":"linux"}
        ────────────────────────────────────────────

```
This is how it looks after the implementation

Reply:
will this be printed every time we get `kepler` or need to pass a flag to `oc` ?

Reply:
No, that will be the default after this patch merges, due to `printcolumn` that is added.